src/App.tsx

```diff
@@ -1,8 +1,77 @@
-import React from 'react';
+import React, {FC, useState} from 'react';
+import { TodoList } from './components/TodoList';
 import './App.css';
+import { getTodos, getUsers } from './api/getData';       
+import { Actions } from './components/Actions';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+export const  App:FC = () => {
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
примитивы не нужно указывать, устанавливаются автоматически
```suggestion
  const [isLoaded, setIsLoaded] = useState(false);
  const [isLoading, setIsLoading] = useState(false);
```